Resources/Maps/_Mono/Shuttles/framework.yml

```
@@ -0,0 +1,541 @@
+grids:
+- 2
+orphans:
+- 2
+nullspace: []
+tilemap:
+  2: Space
+  1: FloorDarkMono
+  0: FloorGlass
+  3: Lattice
+entities:
+- proto: ""
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 2 but found 0

(indentation)